pkg/api/server/register_images.go

```diff
@@ -55,6 +55,27 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//  - images (compat)
 	// summary: List Images
 	// description: Returns a list of images on the server. Note that it uses a different, smaller representation of an image than inspecting a single image.
+	// parameters:
+	//   - name: "all"
+	//     in: "query"
+	//     description: "Show all images. Only images from a final layer (no children) are shown by default."
```

Comment:
Can't we just say "non-dangling images" instead? It seems much clearer

Reply:
thats what you agreed to earlier ...